daemon/src/rollover_taker.rs

```diff
@@ -160,18 +159,18 @@ impl Actor {
         Ok(())
     }
 
-    async fn update_proposal(
-        &self,
-        proposal: Option<(RolloverProposal, SettlementKind)>,
-    ) -> Result<()> {
-        self.projection
+    /// Update the proposal state in the projection actor.
+    ///
+    /// This function is non-fallible because we don't care if the actor goes away, it is not
+    /// critical to continue with the protocol.
```

Comment:
👍 Good learning for the rest of the ephemeral actors.

Reply:
Well, the projection actor should likely be gone from these actors completely!

Reply:
That is the dream!